CMakeLists.txt

```diff
@@ -413,6 +413,9 @@ endif()
 
 if(SENTRY_BACKEND_CRASHPAD)
 	option(SENTRY_CRASHPAD_SYSTEM "Use system crashpad" OFF)
+	if(CRASHPAD_WER_ENABLED)
+		add_compile_definitions(CRASHPAD_WER_ENABLED)
+	endif()
```

Comment:
Moving this here will break the current setup because `CRASHPAD_WER_ENABLED` is automatically set by the `crashpad` build script (if the VS runtime target supports it) and then propagated to the parent scope. If we query it up here, we bypass this detection, and the compile definition is never set even though `WER` is enabled.

Please move this to the end of the branch of `if(SENTRY_BACKEND_CRASHPAD)`.